fastlane/Fastfile

```diff
@@ -581,11 +582,11 @@ lane :rubocop do
   sh('bundle exec rubocop')
 end
 
-desc 'Run source code linting'
-lane :run_linter do
+desc 'Run source code formatting/linting'
+lane :run_swift_format do |options|
   Dir.chdir('..') do
-    UI.error('SwiftFormat lint was skipped. Check out https://github.com/GetStream/ios-issues-tracking/issues/465 for more info')
-    # sh('mint run swiftformat --lint --config .swiftformat Sources --exclude **/Generated, **/generated, **/protobuf, **/OpenApi, Sources/StreamChatSwiftUI/StreamNuke')
+    action = options[:lintOnly] ? "--lint" : ""
```

Comment:
No camel case in Ruby plz 🙈
```suggestion
    action = options[:lint_only] ? "--lint" : ""
```


Or even just `lint`, the other command will just format without linting, won't it?
```suggestion
    action = options[:lint] ? "--lint" : ""
```